internal/githubapi/errors.go

```diff
@@ -95,11 +95,7 @@ func (e *GraphQLErrors) Errors() []GraphQLError {
 
 // Is implements the errors.Is interface.
 func (e *GraphQLErrors) Is(target error) bool {
-	if target == e {
```

Comment:
`target` will never equal `e` because `target == error` and `e == *GraphQLErrors`

Reply:
Good point.